script/deploy/2_ConfigureOFT.s.sol

```diff
@@ -0,0 +1,39 @@
+/* solhint-disable no-console, gas-custom-errors */
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+import {BaseScript} from "script/BaseScript.s.sol";
+
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+
+// forge script ConfigureOFT --rpc-url ${rpc} \
+// --sig "run(string calldata,string calldata)" ${input_path} ${deployment_path} \
+// --account ${deployerAccountName} --sender ${deployer} --broadcast
+
+contract ConfigureOFT is BaseScript {
```

Comment:
Extracts the configure steps outside of the deploy script in a separate `ConfigureOFT` script